### PR TITLE
Update 1907-11-11.xml

### PR DESCRIPTION
**Pull request by onaranjo19**

### CODE REVIEW

**Comment by whanley** [comment]
Lots of good work here. A few things:
- put all material on each page inside a `<div type="page">` pair of tags. I can show you how to do this if it's unclear.
- read your page 3 and check the text--you might have to OCR it differently. The paragraphs need to be there, and in the right order.
- the telegram section in page 5 etc is `section="wire"`, not londonLetter. For details see https://dig-eg-gaz.github.io/contents/features/

**Comment by whanley** [comment]
Great job. Tuesday is complete. Monday will be complete when you tidy it up (following these directions https://dig-eg-gaz.github.io/how-to/digitization/polished-tei-instructions/). Remember to update all templates--I see that you're mostly there. More to add to Wednesday and Thursday obviously.

File: 1907-11-11.xml [hunk 1]
```diff
@@ -33,1532 +32,6264 @@
...
+                            <cell><name>Sachsen</name></cell>
+                            <cell><measure quantity="5026" unit="ton">5026</measure> Tons</cell>
+                            <cell>about <date when="1907-07-31">31 July</date></cell>
+                            <cell/>
+                            <cell/>
+                            <cell/>
+                        </row>
+                        <row role="label">
+                            <cell cols="3">OUTWARD: for CHINA and JAPAN via SUEZ, ADEN, COLOMBO,
+
```
Comment (huoyanzhizhou) [inline 1907-11-11.xml]: Line 1699: cb=1?

Comment (huoyanzhizhou) [inline 1907-11-11.xml]: No header?

Comment (huoyanzhizhou) [inline 1907-11-11.xml]: Line 2133: Before the first cb in a  page, you will need the cols. I also see other small issue about using columns in your file, and you need to check your cb and cols.

Comment (huoyanzhizhou) [inline 1907-11-11.xml]: Line 1171: 1911-11-29? You need to check and update your measures.

File: 1907-11-13.xml [hunk 2]
```diff
@@ -0,0 +1,2544 @@
...
+                            <cell>about <date when="1905-08-14">14 August</date></cell>
+
+                        </row>
+                        <row>
+                            <cell><name>Gera</name></cell>
+                            <cell><measure quantity="5005" unit="ton">5005</measure> Tons</cell>
+                            <cell>about <date when="1905-07-31">31 July</date></cell>
+                            <cell><name>Pr. Regt. Luitpold</name></cell>
+                            <cell><measure quantity="6288" unit="ton">6288</measure> Tons</cell>
+
```
Comment (huoyanzhizhou) [inline 1907-11-13.xml]: Every individual essays in a section should be made as item div under the section div.

**Comment by huoyanzhizhou** [comment]
Good updates. @whanley I think we could merge this one even though there is a new unfinished file.

**Pull request merged**